src/test/java/spoon/test/imports/ImportTest.java

```diff
@@ -1773,4 +1767,72 @@ public void testImportsForElementsAnnotatedWithTypeUseAnnotations() {
 				.filter(ctImport -> ctImport.prettyprint().equals("import spoon.test.imports.testclasses.badimportissue3320.source.other.SomeObjectDto;"))
 				.count());
 	}
+
+	@GitHubIssue(issueNumber = 4786, fixed=false)
```

Comment:
And here the number should be `4783` or?

Reply:
That was my mistake, I picked the wrong issue, sorry :)